packages/walletconnect-solana/package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@jnwng/walletconnect-solana",
-    "version": "0.0.2",
+    "version": "0.0.3",
```

will bump this version separately
```suggestion
    "version": "0.0.2",
```